docs/pages/_document.js

```diff
@@ -51,7 +51,7 @@ export default class MyDocument extends Document {
           */}
           <link rel="manifest" href="/static/manifest.json" />
           {/* PWA primary color */}
-          <meta name="theme-color" content={themeColor} />
+          <meta name="theme-color" content={indigo[500]} />
```

Comment:
indigo[500] is the current default primary.main color

Reply:
Could we pull the value from the default theme instead of hardcoding it?